### ISSUE TITLE
Apache Kafka Multiple Processor not working, Error while fetching metadata with correlation id xx

### ISSUE THREAD

**Issue by MrMoon** [spring-cloud/spring-cloud-stream#2076]
TL:DR
I have multiple functions, processors with an Apache Kafka, that keep giving warning that slows the application down from how much warnings I get,
`
2021-01-08 22:33:10.535  WARN 29114 --- [-StreamThread-1] org.apache.kafka.clients.NetworkClient   : [Consumer clientId=apollo-course-9ece1ae1-68da-442b-8a08-b918b7a71d97-StreamThread-1-consumer, groupId=apollo-course] Error while fetching metadata with correlation id 12 : {apollo-course-KSTREAM-AGGREGATE-STATE-STORE-0000000002-repartition=UNKNOWN_TOPIC_OR_PARTITION}
`
Hi there, 
I was wondering how can I have two processors inside the same application, with an Apache Kafka Clusters
when I did it I got these kind of WARN, 

that slowed the application down because of how much warning there is, 
I try to just work around having only one processor by using BiFunction or stuff like that to have multiple input and then process them inside the same function and produce the output inside the function as well and it worked like a charm, but I still think it's better to separate them in different functions, because of the fact that it's better to handle and can be maintained easier, as well as having a high cohesion and low coupling functions.
I have try to put them in different classes but no progress, the WARN is still there, 

Please tell me if I am doing anything wrong, cause I'm still a beginner in the cloud and stream things,  but I looked and could not figure it out
here is my code, [Apollo-Course](https://github.com/Apollo-LW/course/tree/fix_model), the processor is in the kafka package in a class name CourseProcessors, there is the old single processor commented out as well, 
and the config is in the application.yml file of course 
I did not want to put it here so it's not a long confusing comment/issue
if this is in the wrong place, other spring-cloud project or Stack Overflow, please let me know
THANK YOU IN ADVANCE
